armada/handlers/k8s.py

```diff
@@ -58,3 +58,10 @@ def get_namespace_pod(self, namespace="default"):
         '''
 
         return self.client.list_namespaced_pod(namespace)
+
+    def get_all_pods(self):
+        '''
```

Comment:
So we need to change how we do doc strings @g0r1v3r4 is correct we should be using """.